### ISSUE TITLE
Cannot drap and drop element using xpath although updating taiko lastest version

### ISSUE THREAD

**Issue by levietquynh** [getgauge/taiko#1400]
### Expected behavior
I want to drap and drop an element on HTML5 using xpath 

### Actual behavior
I see source and destination element can be highlighted but it cannot drag and drop.
Note: I can drag drop using text, but nothing happens when I change to xpath

### Steps to reproduce
1. I use await dragAndDrop($(source_xpath), $(destination_xpath));

### Versions
node v12.16.1 and taiko v1.0.16

**Comment by BugDiver** [comment]
@levietquynh The pr #1358 Adds support for drag and drop. It's also released as v1.0.16
Here is a sample which is working for me

The HTML
```html
<!DOCTYPE HTML>
<html>
<head>
<style>
#div1 {
  width: 350px;
  height: 70px;
  padding: 10px;
  border: 1px solid #aaaaaa;
}
</style>
<script>
function allowDrop(ev) {
  ev.preventDefault();
}

function drag(ev) {
  ev.dataTransfer.setData("text", ev.target.id);
}

function drop(ev) {
  ev.preventDefault();
  var data = ev.dataTransfer.getData("text");
  ev.target.appendChild(document.getElementById(data));
}
</script>
</head>
<body>

<p>Drag the W3Schools image into the rectangle:</p>

<div id="div1" ondrop="drop(event)" ondragover="allowDrop(event)">Bar</div>
<br>
<div id="drag1" draggable="true" ondragstart="drag(event)" >Foo</div>

</body>
</html>
```
The Taiko Script:
```js
const { openBrowser, goto, $, dragAndDrop, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto('file://*******/index.html');
        await dragAndDrop($('//*[@id="drag1"]'), $('//*[@id="div1"]'));
    } catch (error) {
        console.error(error);
    } finally {
        await closeBrowser();
    }
})();

```

**Comment by levietquynh** [comment]
I can use Drag and Drop by xpath now, it's so strange because it has never worked before.
I'll close this issue.
